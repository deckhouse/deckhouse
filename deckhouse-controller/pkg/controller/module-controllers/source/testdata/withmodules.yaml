---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: 90f0955ee984feab5c50611987008def
  name: test-source-1
spec:
  registry:
    dockerCfg: YXNiCg==
    repo: dev-registry.deckhouse.io/deckhouse/modules
    scheme: HTTPS
---
apiVersion: deckhouse.io/v1alpha2
kind: ModuleUpdatePolicy
metadata:
  name: test-alpha
spec:
  releaseChannel: Alpha
  update:
    mode: Manual
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: disabledmodule
properties:
  source: test-source-1
  availableSources:
    - test-source-1
status:
  phase: NotInstalled
  conditions:
    - type: EnabledByModuleConfig
      status: "False"
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: enabledmodule
properties:
  source: test-source-1
  availableSources:
    - test-source-1
status:
  phase: NotInstalled
  conditions:
    - type: EnabledByModuleConfig
      status: "True"
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: withpolicymodule
properties:
  source: test-source-1
  updatePolicy: test-alpha
  availableSources:
    - test-source-1
status:
  phase: NotInstalled
  conditions:
    - type: EnabledByModuleConfig
      status: "True"
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: notthissourcemodule
properties:
  source: test-source-2
  availableSources:
    - test-source-1
status:
  phase: NotInstalled
  conditions:
    - type: EnabledByModuleConfig
      status: "True"