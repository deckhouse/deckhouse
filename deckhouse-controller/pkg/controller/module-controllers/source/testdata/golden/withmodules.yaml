---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: 90f0955ee984feab5c50611987008def
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/module-exists
  name: test-source-1
  resourceVersion: "1001"
spec:
  registry:
    ca: ""
    dockerCfg: YXNiCg==
    repo: dev-registry.deckhouse.io/deckhouse/modules
    scheme: HTTPS
status:
  message: ""
  modules:
  - name: disabledmodule
  - checksum: 'sha256:'
    name: enabledmodule
  - name: notthissourcemodule
  - checksum: 'sha256:'
    name: withpolicymodule
    policy: test-alpha
  modulesCount: 4
  syncTime: "2019-10-17T15:33:00Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    release.deckhouse.io/apply-now: "true"
  creationTimestamp: null
  labels:
    module: enabledmodule
    modules.deckhouse.io/update-policy: ""
    release-checksum: 1beb143dffb1b662137094e7faea1e17
    source: test-source-1
  name: enabledmodule-v1.2.3
  ownerReferences:
  - apiVersion: deckhouse.io/v1alpha1
    controller: true
    kind: ModuleSource
    name: test-source-1
    uid: ""
  resourceVersion: "1"
spec:
  moduleName: enabledmodule
  version: 1.2.3
  weight: 900
status:
  approved: false
  message: ""
  pullDuration: 0s
  size: 0
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    release.deckhouse.io/apply-now: "true"
  creationTimestamp: null
  labels:
    module: withpolicymodule
    modules.deckhouse.io/update-policy: test-alpha
    release-checksum: 1beb143dffb1b662137094e7faea1e17
    source: test-source-1
  name: withpolicymodule-v1.2.3
  ownerReferences:
  - apiVersion: deckhouse.io/v1alpha1
    controller: true
    kind: ModuleSource
    name: test-source-1
    uid: ""
  resourceVersion: "1"
spec:
  moduleName: withpolicymodule
  version: 1.2.3
  weight: 900
status:
  approved: false
  message: ""
  pullDuration: 0s
  size: 0
  transitionTime: null
